var/spack/repos/builtin/packages/apple-libuuid/package.py

```diff
@@ -0,0 +1,45 @@
+# Copyright 2013-2020 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class AppleLibuuid(BundlePackage):
+    """Placeholder package for Apple's analogue to non-GNU libuuid"""
+
+    homepage = "https://opensource.apple.com/tarballs/Libsystem/"
+
+    version('1353.100.2')
```

Comment:
We used to have some logic in place that allows packages to not have any `version`. The unit tests seem to expect certain packages to have versions.